chrome/content/components/wzQuicktext.js

```diff
@@ -13,11 +18,6 @@ const kHomepage     = "https://github.com/thundernest/quicktext/wiki/";
 
 function wzQuicktext() {}
 wzQuicktext.prototype = {
```

Comment:
You could just make this `var gQuicktext = { ..`. Given it is a singleton and there is no constructor code you don't need the class overhead

Reply:
Done.